Library/Homebrew/test/caveats_spec.rb

```diff
@@ -26,4 +26,39 @@ def caveats
       expect(described_class.new(f)).not_to be_empty
     end
   end
+
+  describe "#caveats" do
+    describe "when f.plist is not null" do
+      it "when plist_startup is not null" do
```

Comment:
For `it` test names, we want to describe what the test does, not just when it applies. For example, `it "prints plist startup information when plist_startup is not null"`.